### PR TITLE
Generate a single xliff file for each language.

### PR DESCRIPTION
**Pull request by TimBarham**
Rather than one per source file per language. At the request of our loc team.

### CODE REVIEW

File: tools/i18n/update.js [hunk 1]
```diff
@@ -69,22 +72,30 @@ htmlFiles.forEach(function (htmlFile) {
         return previous;
     }, {});
 
-    updateLanguageXlfJson(sourceHtmlRelativePath, strings).then(function (result) {
+    return updateLanguageXlfJson(sourceHtmlRelativePath, strings).then(function (result) {
```
Comment (albertinadx) [inline tools/i18n/update.js]: I would prefer if we can put `then`, `fail` etc callbacks in a new line, to improve readability. That was an implicit convention that we used to follow.
```
return updateLanguageXlfJson(/* ... */)
    .then(function () { 
        // implementation here
    });

```

Comment (albertinad) [inline tools/i18n/update.js]: I would prefer if we can put then, fail etc callbacks in a new line, to improve readability. That was an implicit convention that we used to follow.
```
return updateLanguageXlfJson(/* ... */)
    .then(function () { 
        // implementation here
    });
```

**Comment by albertinad** [comment]
LGTM!

File: tools/i18n/update.js (hunk 1 above)
Comment (TimBarham) [inline tools/i18n/update.js]: Sure, will fix. Though I hate the extra indentation that results. One day we'll move this to TypeScript and use await/async and avoid all this awkward Promise stuff 😄.

**Comment by TimBarham** [comment]
PR auto-closed because I deleted the base branch. Made the commit direct to master.

**Pull request closed without merging**